extensions/amp-story/1.0/amp-story.js

```diff
@@ -481,6 +488,11 @@ export class AmpStory extends AMP.BaseElement {
       this.isLandscapeSupported_() ||
       !this.element.hasAttribute('desktop-aspect-ratio')
     ) {
+      // Store default desktop aspect ratio
+      this.storeService_.dispatch(
```

Comment:
Does this set the aspect ratio to `DEFAULT_DESKTOP_ONE_PANEL_ASPECT_RATIO` even when a story is full screen?

Reply:
No it won't it only sets the --i-amphtml-story-player-panel-ratio on player, which would only be used in desktop mode.

That said, this is not needed. The default ratio is already specified both in story and player's css file. The player one is in here: https://github.com/ampproject/amphtml/blob/main/css/amp-story-player-shadow.css#L87. Good catch!